src/server/src/routes/board/controller.js

```diff
@@ -0,0 +1,15 @@
+'user strict';
+
+let board = {};
+
+board.query = (req, res) => {
+    const BoardService = require('../../services/board/index');
+    return BoardService.prototype.getSingleJob()
+        .then((result) => {
+            res.status(200).json({'response': 'Receiving response from search service: ', result});

```

Comment:
change the response object to just your result object:

res.status(200).json(result);